benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+# Link binary ${basename} against the threading library,
+# and all libraries that are specified as additional arguments.
+function (linkBenchmark basename)
+    target_link_libraries(${basename} absl::flat_hash_map ${ARGN} ${CMAKE_THREAD_LIBS_INIT})
+endfunction()
+
+# Compile the benchmark with MainPrintTimes and link the benchmark, but
+# do not run it.
+# Usage: addAndLinkBenchmarkWithPrintTimes(compileName, cppFileName, [additionalLibraries...])
+#   - compileName : The name, that the compiled end product should have.
+#   - cppFileName : The .cpp file, that contains your benchmark without the file extension.
+function(addAndLinkBenchmarkWithPrintTimes compileName cppFileName)
+    add_executable(${compileName}  Benchmark.cpp "${cppFileName}.cpp" MainPrintTimes.cpp)
+    linkBenchmark(${compileName} ${ARGN})
+endfunction()
```

Comment:
If you have separate `compileName` and `cppFileName` then I would use the filenames including the `.cpp`. We only do this in the tests because there the names are the same.

Reply:
(similar for all the functions below).

Reply:
Done, see [6235fd7](https://github.com/ad-freiburg/qlever/pull/860/commits/6235fd7770174bc8bc0b64ca31112e997c957039).